release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1579,10 +1579,8 @@ This update adds the optional `operatorframework.io/bundle-unpack-min-retry-inte
 [id="ocp-4-15-scalability-and-performance-bug-fixes"]
 ==== Scalability and performance
 
-* When installing using ZTP an install manifest has been added to enable cgroupv1, which prevents an extra reboot at install time. (link:https://issues.redhat.com/browse/OCPBUGS-26540[*OCPBUGS-26540*])
-
-* Previously, a race condition between udev events and the creation queues associated with physical device led to some of the queues being configured with the wrong Receive Packet Steering (RPS) mask when they should be reset to zero. This resulted in the RPS mask being configured on the queues of the physical devices, meaning they were using RPS instead of Receive Side Scaling (RSS), which could impact the performance. With this fix, the event was changed to be triggered per queue creation instead of on device creation. This guarantees that no queue will be missing. The queues of all physical devices are now set up with the correct RPS mask, which is empty. (link:https://issues.redhat.com/browse/OCPBUGS-18662[*OCPBUGS-18662*])
-
+ * Previously, a race condition between `udev` events and the creation queues associated with physical device led to some of the queues being configured with the wrong Receive Packet Steering (RPS) mask while they should be reset to zero. This resulted in the RPS mask being configured on the queues of the physical devices, meaning they were using RPS instead of Receive Side Scaling (RSS), which could impact the performance. With this fix, the event was changed to be triggered per queue creation instead of on device creation. This guarantees that no queue will be missing. The queues of all physical devices are now set up with the correct RPS mask which is empty. (link:https://issues.redhat.com/browse/OCPBUGS-18662[*OCPBUGS-18662*])
```

Comment:
```suggestion
 * Previously, a race condition between `udev` events and the creation queues associated with the physical device led to some of the queues being configured with the wrong Receive Packet Steering (RPS) mask when they should be reset to zero. This resulted in the RPS mask being configured on the queues of the physical devices, meaning they were using RPS instead of Receive Side Scaling (RSS), which could impact the performance. With this fix, the event was changed to be triggered per queue creation instead of at device creation. This guarantees that no queue will be missing. The queues of all physical devices are now set up with the correct RPS mask, which is empty. (link:https://issues.redhat.com/browse/OCPBUGS-18662[*OCPBUGS-18662*])
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#w